tracer/test/Datadog.Trace.Security.Unit.Tests/EventTrackingSdkTests.cs

```diff
@@ -15,6 +15,7 @@
 
 namespace Datadog.Trace.Security.Unit.Tests;
 
+[Collection("SecuritySequentialTests")]
```

Comment:
Do these really need to be run sequentially? I  don't see anything using statics or the WAF? 🤔

Reply:
The main reason are the memory leak tests. If we run in parallel the process memory measurements get corrupted